server/athenian/api/openapi/openapi.yaml

```diff
@@ -5524,82 +5524,126 @@ components:
+        work_began:
+          description: Minimum time when a child issue entered "In Progress" stage.
+            This timestamp can be missing and is always less than or equal to
+            `resolved`.
+          format: date-time
+          type: string
+        lead_time:
+          allOf:
+          - description: Epic's Lead Time.
+          - $ref: '#/components/schemas/TimeDuration'
+        resolved:
+          description: When the epic finished. This timestamp can
+            be missing and is always greater than or equal to `work_began`.
+          format: date-time
+          type: string
         children:
-          description: Details about the contained sub-issues.
+          description: Details about the child issues.
           items:
             $ref: '#/components/schemas/JIRAEpicChild'
           type: array
       required:
-      - children
       - id
       - title
+      - created
       - updated
+      - reporter
+      - comments
+      - status
       type: object
     JIRAEpicChild:
       additionalProperties: false
       description: Brief details about a JIRA issue in an epic.
-      example:
-        id: DEV-157
-        status: Done
-        type: Task
       properties:
         id:
           type: string
-        status:
-          description: Exact name of the issue's status.
+        title:
           type: string
-        type:
+        created:
+          description: When this epic was created.
+          format: date-time
           type: string
+        updated:
+          description: When this epic was last updated.
+          format: date-time
+          type: string
+        reporter:
+          description: Name of the person who reported the issue.
+          type: string
+        assignee:
+          description: Name of the assigned person.
+          type: string
+        comments:
+          description: Number of comments in the issue including sub-tasks.
+          type: integer
+        priority:
+          $ref: '#/components/schemas/JIRAPriority'
+        status:
+          $ref: '#/components/schemas/JIRAStatus'
         work_began:
           description: When the issue entered the "In Progress" stage. This timestamp
             can be missing and is always less than or equal to `resolved`.
           format: date-time
           type: string
+        lead_time:
+          allOf:
+          - description: Epic's Lead Time.
+          - $ref: '#/components/schemas/TimeDuration'
         resolved:
-          description: When the issue was marked as completed. This timestamp can
+          description: When the issue finished. This timestamp can
             be missing and is always greater than or equal to `work_began`.
           format: date-time
           type: string
+        type:
+          description: Name of the issue's type.
+          type: string
+        prs:
+          description: Number of mapped pull requests.
```

Comment:
This is really cool, it's a important asset of the feature. Could we add the overall aggregation to the epic layer? eg epic DEV-245 -> 45 PRs overall

Btw, I assume we talk about all PRs (open and closed), right?

Reply:
Yes, everything that's mapped.